aictl/cli/main.py

```diff
@@ -112,6 +112,36 @@ def ip2p(args):
     print("### saving image files")
     output.images[0].save(args.output_path)
 
+def t2v(args):
+    import torch
+    from diffusers import DiffusionPipeline
+    from diffusers.utils import export_to_video
+
+     # check if on mac and mps is available, fallback to cuda then cpu
+    is_mac = False
+    device = "cuda:0" if torch.cuda.is_available() else "cpu"
+    if torch.backends.mps.is_available():
+        is_mac = True
+        device = torch.device("mps")
+        print("MPS device detected. Using MPS.")
+
+    pipe = DiffusionPipeline.from_pretrained("damo-vilab/text-to-video-ms-1.7b", torch_dtype=torch.float16, variant="fp16")
```

Comment:
Instead of hardcoding the model name here, we can use the `args.model` argument supplied by the user.